### DISCUSSION TITLE
Certificate expired

### DISCUSSION THREAD

**Discussion by frako22** [rubenarslan/formr.org#521] [General]
Hi formr team,

It seems like the certificate of the formr website expired tonight. Unfortunately, there is nothing we can do about it. So I was wondering whether you could fix this. Sorry for the weekend message. 

Thanks for your amazing support!

Best,
Franzi

![image](https://github.com/rubenarslan/formr.org/assets/148468563/eb7eaaa8-609e-47b0-9806-fec592f31933)

**Comment by jkelgs** [comment]
we have the same problem....

**Comment by rubenarslan** [comment]
Cyril updated the certificate now. I don't know why it was not automatically renewed. In the dockerised version of formr that I recommend for self-hosting, certificate renewal is reliably automated.
